src/Courses/TopicGrades/GradeInfoHeader.tsx

```diff
@@ -76,8 +76,10 @@ export const GradeInfoHeader: React.FC<GradeInfoHeaderProps> = ({
         let currentAttemptsCount: number | undefined;
         let currentAverageScore: number | undefined;
         let currentVersionMap: Record<number, Array<number>> = {};
+        let versioned = false;
         if (!_.isNil(workbooks) && !_.isEmpty(workbooks)) {
             const workbookKeys = Object.keys(workbooks);
+            versioned = !_.isNil(Object.values(workbooks)[0].studentGradeInstanceId);
```

Comment:
Do we have to worry about workbooks being an empty object?

Reply:
two lines up, workbooks "not-isEmpty check"